platform/android/java/editor/src/main/java/org/godotengine/editor/BaseGodotEditor.kt

```diff
@@ -122,6 +123,20 @@ open class GodotEditor : GodotActivity() {
 
 	internal open fun getEditorWindowInfo() = EDITOR_MAIN_INFO
 
+	/**
+	 * Set of permissions to be excluded when requesting all permissions at startup.
+	 *
+	 * The permissions in this set will be requested on demand based on use-cases.
```

Comment:
```suggestion
	 * The permissions in this set will be requested on demand based on use cases.
```
Same below for the pre-existing comment with "use-cases".

Reply:
Fixed!